lib/rest/authy/v1/service.d.ts

```diff
@@ -23,13 +23,42 @@ import { SerializableClass } from '../../../interfaces';
  */
 declare function ServiceList(version: V1): ServiceListInstance;
 
+/**
+ * Options to pass to remove
+ *
+ * @property twilioAuthySandboxMode - The twilio_authy_sandbox_mode
+ */
+interface ServiceInstanceDeleteOptions {
+  twilioAuthySandboxMode?: string;
+}
+
+/**
+ * Options to pass to fetch
+ *
+ * @property twilioAuthySandboxMode - The twilio_authy_sandbox_mode
+ */
+interface ServiceInstanceFetchOptions {
+  twilioAuthySandboxMode?: string;
+}
+
+/**
+ * Options to pass to remove
+ *
+ * @property twilioAuthySandboxMode - The twilio_authy_sandbox_mode
+ */
+interface ServiceInstanceRemoveOptions {
```

Comment:
I don't see this used, and it mirrors `ServiceInstanceDeleteOptions`.

Reply:
Fixed this issue globally